.jscs.json

```diff
@@ -0,0 +1,88 @@
+{
```

Comment:
Is this file supposed to be in here?  I assume it's a config for the javascript style checker, but there's no documentation or indication as to what it's for....  did you mean to include it?